setup.py

```diff
@@ -32,7 +32,8 @@
         "fastapi_utils",
         "opencv-python-headless",
         "psutil",
-        "multilingual_clip"
+        "multilingual_clip",
```

Comment:
Maybe we should stop updating setup.py. It's obsolete now, since Marqo is no longer a PyPI package right? cc @farshidz @pandu-k

Reply:
Yeah that makes sense. Maybe we should still update it now, and remove the whole thing in the future?

Reply:
That's fine with me